src/components/Modal/Modal.tsx

```diff
@@ -38,7 +80,7 @@ export const Modal = (props: ModalProps) => {
     title,
     // description,
     children,
-    actions = [],
+    // actions = [],
```

Comment:
@AaronPlave was this an okay change? I wasn't sure where `actions` was supposed to be used.

Reply:
Oh it looks like actions and description aren't used at all, can get rid of those.